src/index.html

```diff
@@ -1,14 +1,27 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <div class="wrap">
+      <div class="desc">
+        Я - секундомер! Хочешь остановить меня? Наведи курсор на меня!
+      </div>
+      <div class="watch">
+        <div class="miniWatch">
+          <div class="miniWatch__seconds"></div>
+          <div class="miniWatch__sircle"></div>
+          <div class="miniWatch__seconds"></div>
```

Comment:
```suggestion
        <div class="mini-watch">
          <div class="mini-watch__seconds"></div>
          <div class="mini-watch__sircle"></div>
          <div class="mini-watch__seconds"></div>
```
краще притримуватись узгоджених схем найменування БЕМ
https://en.bem.info/methodology/naming-convention/

Reply:
Минутную стрелку добавил, БЭМ исправил